articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 Original Author: {{ page.author }}
 
 
+## Preface
+
+ROS is developed with a federated model.
+While this comes with a lot of advantages it makes building a set of interdependent packages more complex.
+
+Commonly a user has to build interdependent packages in the correct topological order.
+And for each package the steps include reading the documentation of a package, then building and installing it following a few build system specific commands and then setting up the environment for other packages to use it.
+For a larger number of packages this approach does not scale well.
+
+ROS mitigates that complexity with automation in the build process.
+Wherever possible ROS uses existing, standard tools, e.g. CMake and Python setuptools.
```

Comment:
You could probably drop the `existing, ` since that's sort of redundant to the idea of a standard tool.